david_laster/midterm/lib/client.rb

```diff
@@ -0,0 +1,14 @@
+class Client
+	attr_accessor :name, :age, :pets
+	def initialize(name,age)
+		@name = name
+		@age = age.to_i
+		@pets = []
+	end
+
+	def to_s
+		"Name: #{@name} Age: #{@age}"
+	end
```

Comment:
maybe for the `Client' print method, it would be cool to print different statements depending on if the client has pets or not?

```
def to_s
    if @pets.any?
        if @pets.count == 1
            "#{@name} is #{@age} and has #{@pets.count} pet. It's name is #{@pets.first.name}"
        else
	     "#{@name} is #{@age} and has #{@pets.count} pets. Their names are #{(@pets.map { |pet| pet.name }).join(", ")}"
        end
    else
        "#{@name} is #{@age}"
    end
end
```